BUILDING.md

```diff
@@ -0,0 +1,21 @@
+# Building `authsae`
+
+`authsae` and its associated library `libsae` are easy to build on Linux. This page is intended as a quick reference on how to start building it from scratch on an Ubuntu VM, including which dependencies likely need to be installed.
+
+1. Set up an Ubuntu VM (these instructions assume 18.04.1 LTS).
+    1. `sudo apt install open-vm-tools-desktop` – gets you copy/paste stuff, at least for VMware
+    1. `sudo apt install net-tools` – gets you `ifconfig`
+    1. Reboot
+1. Install all the things:
+    ```
+    sudo apt install gcc
```

Comment:
You can put all of these on one line -- sudo apt install gcc make cmake pkg-config ...

And installing all those deps is relevant for building on a linux desktop also (at least debian/ubuntu).  So, maybe make the VM stuff a different section? 

I think it makes sense to put all of it in the README.md rather than in a separate file.

Reply:
Sure, can do, one sec.